src/main/java/io/codelex/arithmetic/practice/Exercise2.java

```diff
@@ -0,0 +1,27 @@
+package io.codelex.arithmetic.practice;
+import java.util.Scanner;
+public class Exercise2 {
+    public static void main(String[] args){
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Enter a number");
+        int number = scanner.nextInt();
+
+        if (isEven(number)) {
```

Comment:
Faktiski ja nav isEven, tad var likt pēc tam else. Vari atstāt kā ir, bet iespējams šis ir tas gadījums, kad nest ārā atsevišķas metodes isEven un isOdd ir nedaudz too much. :D